src/model/input.ts

```diff
@@ -8,129 +12,212 @@ const core = require('@actions/core');
  * Note that input is always passed as a string, even booleans.
  */
 class Input {
+  public static cliOptions;
+  public static githubInputEnabled: boolean = true;
+
+  // also enabled debug logging for cloud runner
+  static get cloudRunnerTests(): boolean {
+    return Input.getInput(`cloudRunnerTests`) || Input.getInput(`CloudRunnerTests`) || false;
+  }
+  private static getInput(query) {
+    const coreInput = core.getInput(query);
+    if (Input.githubInputEnabled && coreInput && coreInput !== '') {
+      return coreInput;
+    }
+
+    return Input.cliOptions !== undefined && Input.cliOptions[query] !== undefined
+      ? Input.cliOptions[query]
+      : process.env[query] !== undefined
+      ? process.env[query]
+      : process.env[Input.ToEnvVarFormat(query)]
+      ? process.env[Input.ToEnvVarFormat(query)]
+      : '';
+  }
+  static get region(): string {
+    return Input.getInput('region') || 'eu-west-2';
+  }
+  static async githubRepo() {
+    return (
+      Input.getInput('GITHUB_REPOSITORY') ||
+      Input.getInput('GITHUB_REPO') ||
+      (await GitRepoReader.GetRemote()) ||
+      'game-ci/unity-builder'
+    );
+  }
+  static async branch() {
+    if (await GitRepoReader.GetBranch()) {
```

Comment:
Note 1 for later: Methods and function definition should always have a space between them for readability.

We should probably add this as an ESLint rule. (also fine to just create an issue for it)

Reply:
Created an issue